src/zoid/axo/component.jsx

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+/** @jsx node */
+/* eslint max-lines: 0 */
+
+import { loadConnectScript } from "@paypal/connect-loader-component";
+import { getClientID, getClientMetadataID } from "@paypal/sdk-client/src";
+// eslint-disable-next-line flowtype/no-weak-types
+export type AXOComponent = any;
+
+export function getAxoComponent(): AXOComponent {
+  const cmid = getClientMetadataID();
+  const clientID = getClientID();
+  // this will change to whatever options we received
+  // from the merchant
+  try {
+    return loadConnectScript({ cmid, clientID });
```

Comment:
We'd need to also retrieve the user Id token using the getter defined in sdk-client and pass it in. 

We want to `await` the script loading so we know it's done, then a window object is available for us to initialize. We then pass the appropriate params.
```suggestion
    await  loadConnectScript();
    window.braintree.connect.create({ cmid, clientID, userIdToken })
```